File: java/utility/MakeContext.java
package voxgig.bluefindecryptxsdk.utility;

import java.util.Map;

import voxgig.bluefindecryptxsdk.core.Context;

final class MakeContext {

  private MakeContext() {}

  static Context makeContext(Map<String, Object> ctxmap, Context basectx) {
    return new Context(ctxmap, basectx);
  }
}

File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// BluefinDecryptx SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class BluefinDecryptxMakeContext
{
    public static function call(array $ctxmap, ?BluefinDecryptxContext $basectx): BluefinDecryptxContext
    {
        return new BluefinDecryptxContext($ctxmap, $basectx);
    }
}
